css/properties/-webkit-line-clamp.json

```
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "-webkit-box-reflect": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/-webkit-line-clamp",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Based upon the Safari version you've reported, this would be landing in Chrome 6.

```suggestion
              "version_added": "6"
```